src/gleescript.gleam

```diff
@@ -121,7 +142,12 @@ fn load_config() -> snag.Result(Config) {
     |> snag.context("Failed to get package name from gleam.toml"),
   )
 
-  Ok(Config(package_name: package_name))
+  let target_dir = case tom.get_string(config, ["build", "target-dir"]) {
+    Ok(target) -> "./" <> target <> "/"
+    Error(_) -> "./"
+  }
```

Comment:
We'll need to support absolute directories and trailing slashes here.

Reply:
noted.